src/schemas/json/staticwebapp.config.json

```diff
@@ -620,25 +570,21 @@
                 ".*": {
                     "oneOf": [
                         {
-                            "allOf": [
-                                {
-                                    "$ref": "#/definitions/routeRewrite"
+                            "type": "object",
+                            "properties": {
+                                "redirect": {
+                                    "type": "string",
+                                    "description": "Redirect to a relative or absolute path, or an external URI. Default status code is 302, override with 301."
                                 },
-                                {
-                                    "properties": {
-                                        "statusCode": {
-                                            "type": "integer",
-                                            "description": "Status code for the response",
-                                            "examples": [
-                                                200
-                                            ]
-                                        }
-                                    }
+                                "statusCode": {
```

Comment:
This is for response overrides. None of these are required, but this can probably be improved so that you have to have one of `redirect` or `rewrite`, but only `redirect` can have `statusCode`. I couldn't figure out how to do this.

For `routes` the schema requires at least a `route` property.